README.md

```diff
@@ -12,15 +14,37 @@ The input to the pipeline is a standard sample sheet (passed as `--input samples
 | ------- | ----------------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- |
 | SampleA | sampleA.mlst.json | meta1      | meta2      | meta3      | meta4      | meta5      | meta6      | meta7      | meta8      |
 
-The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/). Details on the columns can be found in the [Full samplesheet](docs/usage.md#full-samplesheet) documentation.
 
 # Parameters
 
 The main parameters are `--input` as defined above and `--output` for specifying the output results directory. You may wish to provide `-profile singularity` to specify the use of singularity containers and `-r [branch]` to specify which GitHub branch you would like to run.
 
 ## Metadata
 
-In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified.
+In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified. These parameters are used to re-name the headers in the final metadata table from the defaults (e.g., rename `metadata_1` to `country`).
+
+## Profile dists
+
+The following can be used to adjust parameters for the [profile_dists][] tool.
+
+- `--pd_outfmt`: The output format for distances, either _matrix_ or _pairwise_.
+- `--pd_distm`: The distance method/unit, either _hamming_ or _scaled_.
+- `--pd_missing_threshold`: The maximum proportion of missing data per locus.
+- `--pd_sample_quality_threshold`: The maximum proportion of missing data per sample.
+- `--pd_match_threshold`: Threshold for matches. Should match unit used in pd_distm. Used only with `--pd_outfmt pairwise`.
```

Comment:
I'm not sure if someone familiar with the problem would know this, but is this an integer for hamming and a float for scaled? Is scaled a percentage threshold (like BLAST percent identity) or is it any number greater than zero?

Reply:
This parameter was removed in 49a9266cb4317b907b4d2c1d5e0bb04e0a47433f

I also discovered it wasn't being passed to profile_dists anyways.